features/unit/v2indexerclient_paths.feature

```diff
@@ -107,27 +107,28 @@ Feature: Indexer Client v2 Paths
-      | /v2/transactions?currency-greater-than=0&tx-type=acfg                                                       |                                                            |               | acfg   |         |                                                      | 0     | 0        | 0        | 0     |                         |                         | 0                   | 0                | 0     |             |                |
-      | /v2/transactions?currency-greater-than=0&txid=TDIO6RJWJIVDDJZELMSX5CPJW7MUNM3QR4YAHYAKHF3W2CFRTI7A          |                                                            |               |        |         | TDIO6RJWJIVDDJZELMSX5CPJW7MUNM3QR4YAHYAKHF3W2CFRTI7A | 0     | 0        | 0        | 0     |                         |                         | 0                   | 0                | 0     |             |                |
+      | path                                                                                                        | account                                                    | notePrefixB64 | txType | sigType | txid                                                 | round | minRound | maxRound | limit | beforeTime              | afterTime               | currencyGreaterThan | currencyLessThan | index | addressRole | excludeCloseTo | groupid                                      | 
+      | /v2/transactions?currency-greater-than=0                                                                    |                                                            |               |        |         |                                                      | 0     | 0        | 0        | 0     |                         |                         | 0                   | 0                | 0     |             |                |                                              |
+      | /v2/transactions?address=PNWOET7LLOWMBMLE4KOCELCX6X3D3Q4H2Q4QJASYIEOF7YIPPQBG3YQ5YI&currency-greater-than=0 | PNWOET7LLOWMBMLE4KOCELCX6X3D3Q4H2Q4QJASYIEOF7YIPPQBG3YQ5YI |               |        |         |                                                      | 0     | 0        | 0        | 0     |                         |                         | 0                   | 0                | 0     |             |                |                                              |
+      | /v2/transactions?address-role=sender&currency-greater-than=0                                                |                                                            |               |        |         |                                                      | 0     | 0        | 0        | 0     |                         |                         | 0                   | 0                | 0     | sender      |                |                                              |
```

Comment:
This is obviously outside the scope of this PR, but why do we always include `currency-greater-than`?